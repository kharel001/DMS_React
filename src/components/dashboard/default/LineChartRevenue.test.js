import React from 'react';
import LineChartRevenue from './LineChartRevenue';
import {
    configure,
    shallow,
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})

const setup = (props = {}) => {
    return shallow( < LineChartRevenue {...props}  ><div></div><div></div><div></div></LineChartRevenue>
    )
}

const findByTestAttr=(wrapper, val)=>{
    return wrapper.find(`[data-test='${val}']`)
}


describe("LineChartRevenue Comp",()=>{
it("Snapshot",()=>{
    const wrapper= setup();
    expect(wrapper).toMatchSnapshot()

})

})