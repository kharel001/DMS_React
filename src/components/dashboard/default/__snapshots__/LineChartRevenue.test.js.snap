// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineChartRevenue Comp Snapshot 1`] = `ShallowWrapper {}`;
